src/main/java/com/diffblue/deeptestutils/powermock/WhiteboxImpl.java

```diff
@@ -0,0 +1,206 @@
+package com.diffblue.deeptestutils.powermock;
+
+import org.powermock.reflect.exceptions.TooManyMethodsFoundException;
+
+import java.lang.reflect.Method;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+// Copyright 2016-2018 Diffblue limited. All rights reserved.
+
+/**
+ * <code>WhiteboxImpl</code> extends the original class in PowerMock
+ * and fixes the issues below.
+ * - TG-5254
+ *
+ * @author <a href="http://diffblue.com">Diffblue</a>
+ */
+public class WhiteboxImpl extends org.powermock.reflect.internal.WhiteboxImpl {
```

Comment:
import org.powermock.reflect.internal.WhiteboxImpl

Reply:
Done and renamed to `DTUWhiteboxImpl`.